src/components/UserPlan.tsx

```diff
@@ -106,27 +98,13 @@ export const UserPlan = ({ user }: UserPlanProps) => {
               credits={maxSubscribers}
               description={description}
             />
-            {!!Object.keys(sms).length && smsEnabled && (
-              <BuyContainer>
-                <p>
-                  <strong>
-                    US${" "}
-                    <FormattedNumber
-                      // eslint-disable-next-line react/style-prop-object
-                      style="decimal"
-                      value={smsRemainingCredits}
-                      minimumFractionDigits={2}
-                      maximumFractionDigits={2}
-                    />
-                  </strong>{" "}
-                  {smsDescription}
-                </p>
-                {smsButtonUrl && (
-                  <a className="user-plan" target="_self" href={smsButtonUrl}>
-                    {smsButtonText}
-                  </a>
-                )}
-              </BuyContainer>
+            {sms && (
+              <SmsInformation
+                buttonText={sms.buttonText}
+                buttonUrl={sms.buttonUrl}
+                description={sms.description}
+                remainingCredits={sms.remainingCredits}
+              />
             )}
```

Comment:
Cómo hablamos, no tengo opinión fuerte, solo lo comento para que quede de referencia.

Si conservaramos el [modelo anterior](https://github.com/FromDoppler/doppler-menu-mfe/pull/173/files#diff-6bfea2ec3c1d21c90d6e1ecd704f380a6eeb44521e1da1a87b6a48ea691b6a43L42-R49), esto podría hacerse de la siguiente manera:

```suggestion
            {!sms.smsEnabled ? null : (
              <SmsInformation
                buttonText={sms.buttonText}
                buttonUrl={sms.buttonUrl}
                description={sms.description}
                remainingCredits={sms.remainingCredits}
              />
            )}
```